intelmq/lib/bot.py

```diff
@@ -245,7 +265,11 @@ def acknowledge_message(self):
         self.last_message = self.current_message
         self.source_pipeline.acknowledge()
 
-    def dump_message(self):
+    def dump_message(self, error_traceback):
+        if self.current_message is None:
```

Comment:
I raised this question because it changes this behavior. As the traceback is also in the logs, it can make sense to dump nothing.

Currently it's handier to inspect the traceback using the intelmqdump tool instead of filtering the logs for tracebacks.